### PR TITLE
Allow webpack 5 as peer dependency

### PR DESCRIPTION
**Pull request by jorrit**
This PR contains:

- [x] bugfix
- [ ] feature
- [ ] refactor
- [ ] documentation
- [ ] other

Are tests included?

- [ ] yes (_bugfixes and features will not be merged without tests_)
- [x] no

Breaking Changes?

- [ ] yes (_breaking changes will not be merged unless absolutely necessary_)
- [x] no

List any relevant issue numbers: #270

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "lint:js": "eslint --fix --cache lib test",
     "lint:json": "prettier --write codecov.yml .circleci/config.yml .eslintrc",
     "lint:package": "prettier --write package.json --plugin=prettier-plugin-package",
-    "posttest": "node set-webpack-version.js \"^4.44.2\" && npm install",
+    "posttest": "node set-webpack-version.js \">=4.44.2\" && npm install",
```
Comment (shellscape) [inline package.json:24]: please revert this change

File: package.json [hunk 2]
```diff
@@ -33,7 +33,7 @@
     "LICENSE"
   ],
   "peerDependencies": {
-    "webpack": "^4.44.2"
+    "webpack": ">=4.44.2"
```
Comment (shellscape) [inline package.json:36]:
```suggestion
    "webpack": "^4.44.2 || ^5.46.0"
```

Using `>=` would allow v6 if/when that happens, which is not correct

**Review by shellscape — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (jorrit) [reply package.json:24]: Should this become '^4.44.2 || ^5.46.0' just like the other setting? Because if I simply revert it, testing will alter package.json and perhaps cause committing unwanted changes.

File: package.json (hunk 2 above)
Comment (jorrit) [reply package.json:36]: Sure thing. My initial solution was to revert to the previous setting, but `^4.44.2 || ^5.46.0` is better.

File: package.json (hunk 1 above)
Comment (shellscape) [reply package.json:24]: There's a larger fix here that's needed related to the set-webpack-version script to prevent bad overwrites. I'll do that after your PR.

Comment (jorrit) [reply package.json]: Cool

**Review by shellscape — APPROVED** [review]

**Comment by jorrit** [comment]
This PR was reverted by 830a1b20d0de5e661b0f51cef62c0afa0945700b.

**Comment by shellscape** [comment]
It actually wasn't. It was reverted by the previous commit, and reinstated by the commit you referenced.

**Comment by jorrit** [comment]
Sorry, I shouldn't have commented before the third coffee in the morning. My apologies.
